### PR TITLE
Migrate tasks to laravel scheduler

### LINKED ISSUE #1558: Migrate Crons To Scheduler
Time Expression | Human Form | Command
-- | -- | --
0 22 * * * | At every 22nd Hour | feedback:summarise --interval=p1d
30 23 * * * | At 23:30 | Sync:Helpdesk
0 23 * * * | At 23:00 | Sync:Mentors
1 0 * * * | At 00:01 | Sync:Community
1 0 * * * | At 00:01 | Sync:RTS
1 0 * * * | At 00:01 | Sync:Moodle
0 2,14 * * * | At 02:00 and 14:00 | Members:CertImport --full
30 */2 * * * | At 30 minutes past every 2 hours | Members:CertImport
45 0 * * * | At 00:45 | Members:CertUpdate --type=daily 5000
15 1 * * 1 | At 01:15 every Monday | Members:CertUpdate --type=weekly 5000
45 1 1 * * | At 01:45 on the 1st of the Month | Members:CertUpdate --type=monthly 5000
45 */2 * * * | At 45 minutes past every 2nd hour. | Members:CertUpdate --type=all 100
/* * * * * | Every Minute | visit-transfer:cleanup
/* * * * * | Every Minute | teaman:runner -v
*/2 * * * * | Every 2nd Minute | networkdata:download
0 * * * * | Every Hour | slack:manager
1 0 * * * | At 00:01 | sys:statistics:daily
30 0 * * * | At 00:30 | visittransfer:statistics:daily

### PR DESCRIPTION
**Pull request by AxonC**
Closes #1558 . Requires disabling of cron jobs on hydra.

### CODE REVIEW

File: app/Console/Kernel.php [hunk 1]
```diff
@@ -28,6 +32,87 @@ protected function schedule(Schedule $schedule)
     {
         $schedule->command('telescope:prune')->daily();
         // $schedule->command('sync:tg-forum-groups')->dailyAt('04:00');
+
+        $schedule->command('sys:statistics:daily')
+            ->dailyAt('00:01')
+            ->onFailure($this->failureMessage('Syncing of daily statistics has failed (sync:statistics:daily)'))
+            ->emailOutputOnFailure($this->failureEmail);
+
+        $schedule->command('sync:helpdesk')
```
Comment (atoff) [inline app/Console/Kernel.php]: Helpdesk should already be covered - ref https://github.com/VATSIM-UK/core/blob/master/app/Listeners/Mship/SyncSubscriber.php#L24

Comment (AxonC) [reply app/Console/Kernel.php]: removed!

**Comment by CalumTowers** [comment]
On hold until we can release and disable crons at the same time.

File: app/Console/Kernel.php [hunk 2]
```diff
@@ -2,11 +2,15 @@
...
 namespace App\Console;
 
+use App\Libraries\Slack;
+use Closure;
 use Illuminate\Console\Scheduling\Schedule;
 use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
 
 class Kernel extends ConsoleKernel
 {
+    private $failureEmail = 'privileged-access@vatsim-uk.co.uk';
```
Comment (CalumTowers) [inline app/Console/Kernel.php]: @AxonC Just realised this - could you update the email to be .uk rather than -uk.co.uk?

**Pull request merged**